mmv1/products/compute/terraform.yaml

```diff
@@ -176,16 +176,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       cdnPolicy.clientTtl: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
```

Comment:
I put [this together](https://github.com/GoogleCloudPlatform/magic-modules/blob/64af16318d8a391050b0c157fc3d38c5cc3431a5/mmv1/products/billingbudget/terraform.yaml#L116) for a different resource that might help.

I had to remove the `default_from_api` field, and then the custom diff suppress will suppress diffs if a specified field is present, or if the value is set to a specific (default) value. This requires putting the default value in the terraform.yaml file vs accepting it from the API, but seems like a worthy tradeoff for backwards compatibility. Diff suppress function is: mmv1/templates/terraform/constants/billing_budget.erb